src/Prometheus.Client.MetricPusher.HostedService.Tests/Prometheus.Client.MetricPusher.HostedService.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
For Prometheus.Client.MetricPusher.HostedService.csproj Nullable is disabled. Enable Nullable in Prometheus.Client.MetricPusher.HostedService.csproj or remove Nullable from test project.

Reply:
Removed